### PR TITLE
Maintain target board for NEUTRONRC

### PR DESCRIPTION
**Pull request by Linjieqiang**
Knowing that in the new version v4.5 starts to use a brand new macro definition build system, so as one of the maintainers of the NEUTRONRC brand, I will fix the past bugs, it has been tested.

Here is the file for local offline build:
[Tatget V4.5.zip](https://github.com/betaflight/config/files/12454895/Tatget.V4.5.zip)

### CODE REVIEW

File: configs/NEUTRONRCF411AIO/config.h [hunk 1]
```diff
@@ -26,17 +26,17 @@
...
 #define USE_GYRO_SPI_ICM42688P
-#define USE_GYRO_SPI_ICM42605
-#define USE_GYRO_SPI_ICM20689
-#define USE_ACC_SPI_ICM20689
-#define USE_ACC_SPI_MPU6000
 #define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACCGYRO_BMI270
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACC_SPI_MPU6000
```
Comment (haslinghuis) [inline configs/NEUTRONRCF411AIO/config.h:35]: Where are the gyro IMU defines for these accelerometers types?

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/NEUTRONRCF411AIO/config.h (hunk 1 above)
Comment (nerdCopter) [reply configs/NEUTRONRCF411AIO/config.h:35]: need:
```
#define USE_GYRO_SPI_ICM42688P
#define USE_GYRO_SPI_MPU6000
```

File: configs/NEUTRONRCF7AIO/config.h [hunk 2]
```diff
@@ -28,7 +28,10 @@
 
 #define USE_ACC
 #define USE_ACC_SPI_ICM20689
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
 #define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
 #define USE_GYRO_SPI_ICM20689
 #define USE_ACCGYRO_BMI270
```
Comment (haslinghuis) [inline configs/NEUTRONRCF7AIO/config.h:36]:
```suggestion
#define USE_ACC
#define USE_ACC_SPI_ICM20689
#define USE_ACC_SPI_MPU6000
#define USE_ACC_SPI_ICM42688P
#define USE_GYRO
#define USE_GYRO_SPI_MPU6000
#define USE_GYRO_SPI_ICM20689
#define USE_GYRO_SPI_ICM42688P
#define USE_ACCGYRO_BMI270
```

File: configs/NEUTRONRCH743AIO/config.h [hunk 3]
```diff
@@ -24,6 +24,15 @@
 #define BOARD_NAME        NEUTRONRCH743AIO
 #define MANUFACTURER_ID   NERC
 
+#define USE_GYRO
+#define USE_ACC
+#define USE_ACCGYRO_BMI270
+#define USE_FLASH
+#define USE_FLASH_W25M02G
```
Comment (haslinghuis) [inline configs/NEUTRONRCH743AIO/config.h:31]: The W25M02G does not perform well in Betaflight. Might consider other options.

File: configs/NEUTRONRCH7BT/config.h [hunk 4]
```diff
@@ -24,12 +24,14 @@
...
 #define MANUFACTURER_ID   NERC
 
-#define USE_ACC
 #define USE_GYRO
+#define USE_ACC
 #define USE_ACCGYRO_BMI270
-#define USE_BARO
-#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25M02G
```
Comment (haslinghuis) [inline configs/NEUTRONRCH7BT/config.h:31]: Same as above

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/NEUTRONRCF411AIO/config.h [hunk 5]
```diff
@@ -26,17 +26,17 @@
 #define BOARD_NAME        NEUTRONRCF411AIO
 #define MANUFACTURER_ID   NERC
 
-#define USE_ACC
-#define USE_ACC_SPI_ICM42688P
-#define USE_ACC_SPI_ICM42605
 #define USE_GYRO
 #define USE_GYRO_SPI_ICM42688P
```
[the remark this answers is not in the document]
Comment (haslinghuis) [reply configs/NEUTRONRCF411AIO/config.h:30]: Yes correct.

**Review by nerdCopter — APPROVED** [review]
approving, seems good; i did not review every detail

File: configs/NEUTRONRCF7AIO/config.h (hunk 2 above)
Comment (haslinghuis) [reply configs/NEUTRONRCF7AIO/config.h:36]: #define USE_GYRO_SPI_ICM42688P is still missing :)

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**